.gitignore

```diff
@@ -6,4 +6,5 @@ hc.sqlite
 hc/local_settings.py
 static-collected
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+.vscode
```

Comment:
You can have a global gitignore to avoid adding obvious files to every projects gitignore.